module/User/src/User/Permissions/Assertion/IsBoardMember.php

```diff
@@ -58,6 +58,6 @@ protected function isCurrentBoard(BoardMember $boardMember)
     {
         $now = new \DateTime();
         return $boardMember->getInstallDate() <= $now &&
-            (null === $boardMember->getDischargeDate() || $boardMember->getDischargeDate >= $now);
+            (null === $boardMember->getDischargeDate() || $boardMember->getDischargeDate() >= $now);
```

Comment:
I didn't follow the discussion in chat, so please verify that removing this bug an improvement.

Reply:
Currently, old board members keep their admin privileges for eternity. This update removes the privileges after the discharge date. This means that they lose them at some point, but still some time after they don't need them anymore.
It may not be the best solution, but it is an improvement from the status quo and fixes a bug.